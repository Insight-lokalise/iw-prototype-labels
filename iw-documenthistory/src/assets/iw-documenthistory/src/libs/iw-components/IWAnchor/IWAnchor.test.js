import React from 'react'
import { render } from '@testing-library/react'
import { IWAnchor } from './IWAnchor'

describe('<IWAnchor>', () => {
  it('should render children', () => {
    const {asFragment} = render(
      <IWAnchor className="B__E--M">
        Clickable
        <strong>text</strong>
      </IWAnchor>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
