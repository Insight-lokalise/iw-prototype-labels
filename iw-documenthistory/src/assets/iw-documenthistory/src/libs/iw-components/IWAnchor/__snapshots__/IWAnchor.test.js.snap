// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IWAnchor> should render children 1`] = `
<DocumentFragment>
  <a
    class="B__E--M"
  >
    Clickable
    <strong>
      text
    </strong>
  </a>
</DocumentFragment>
`;
